### DISCUSSION TITLE
Smoke exhaust in sloping roof

### DISCUSSION THREAD

**Discussion by Picia1994** [firemodels/fds#13063] [Q&A]
Hi,

Is there a possibility to built smoke exhaust in sloping roof? How to do it?

![roof model](https://github.com/firemodels/fds/assets/172837092/4084e680-1ffd-4b8f-8592-f60567953c9a)

**Comment by marcosvanella** [comment]
Hi @Picia1994, what does your smoke exhaust system look like? Are you modeling the sloped roof with GEOM or OBST?

**Comment by Picia1994** [comment]
OBST

**Comment by drjfloyd** [comment]
For pyrosim I think the easiest thing to do would be put a hole in your sloped roof and draw another sloped obstruction inside that hole (set the properties to not allow a hole for that second obstruction). Then you can apply an exhaust surface to the bottom.
